app/talk/latest-comment-link.cjsx

```diff
@@ -9,15 +9,10 @@ merge = require 'lodash.merge'
 
 PAGE_SIZE = require('./config').discussionPageSize
 
-truncate = (string, ending = '', length = 80) ->
+truncate = (string = '', ending = '', length = 80) ->
   return string if string.trim().length <= length
   string.trim().slice(0, (length - ending.length)) + ending
 
-latestCommentText = (discussion) ->
```

Comment:
This was removed because mutating the DOM during a render was doing some really weird stuff.

In some cases, it would cause a re-render infinite loop